### DISCUSSION TITLE
Bard Text Wrapping

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#7321] [Troubleshooting]
I'm trying to set up a bard section where my client can enter text and images in 1 block but have the image wrap the text. However I can't seem to achieve this.

I've tried the tiny mce plugin but don't really want to use it unless I have to.

Is there any way I can wrap text around an image using a bard?

**Comment by edalzell** [comment]
Make a Bard set with an image and a textarea and handle it in the template?
